docs/src/examples/components/Avatar/Variations/AvatarExampleExcludedInitials.shorthand.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react'
 import { Avatar } from '@stardust-ui/react'
 
+const Available = {
```

Comment:
We should probably call this `status` or `availableStatus` to follow code conventions.  Uppercase names are unique to classes.